_inc/client/state/initial-state/reducer.js

```diff
@@ -256,6 +256,17 @@ export function showBackups( state ) {
 	return get( state.jetpack.initialState.siteData, 'showBackups', true );
 }
 
+/**
+ * Check if the site is part of a Multisite network.
+ *
+ * @param {object} state Global state tree
+ *
+ * @return {boolean} True if the site is part of a Multisite network.
+ */
+export function isMultisite( state ) {
+	return get( state.jetpack.initialState.siteData, 'isMultisite', true );
```

Comment:
I think that the default should be `false` here.

```suggestion
	return get( state.jetpack.initialState.siteData, 'isMultisite', false );
```